app/commands/v2/represent_downstream.rb

```diff
@@ -48,7 +48,7 @@ def downstream_live(content_id, locale)
             content_id: content_id,
             locale: locale,
             payload_version: event.id,
-            message_queue_update_type: "links",
+            message_queue_event_type: "links",
```

Comment:
We've spoken to the Search team and they would like to leave this as it is for the moment.